content/getting-started/registering.md

```diff
@@ -4,8 +4,67 @@ prev: "/getting-started/"
 next: "/getting-started/setting-up/"
 ---
 
-# Registering
-
 <!--
-TODO Briefly explain how to create an account and link to https://www.codewars.com
+Steps
+1. Sign up
+2. Choose language
+3. Multiply CAPTCHA
+4. Enlist
+ - confirmation email
+5. Initial training setup
+6. Initial dashboard, popup "Let's get started!", read more
 -->
+
+# Registering
+
+So you decided to join? That's great! There's only a couple of steps to create new account and start solving tasks, learn, and improve your skills!
+
+## First step
+
+First, you need to go to [Sign Up](https://www.codewars.com/join) page, if you haven't yet. You will be presented with a board of many languages to choose from:
+
+![languages board](./img/registering_01_choose_language.png)
+
+Choose whatever language you like, you want to learn, or are familiar with. I want to improve my skills with functional programming, and I already know a bit if Java, so I choose Scala as my initial language. Here you can pick any language available, your selection does not determine anything yet, and you will be always able to choose another language later, if you wish.
+
+## Your first challenge
+
+After selecting the language, you are presented with your very first challenge. It's an entry level task which you need to solve to prove your dedication and commitment to learning, and prove yourself worthy of training among CodeWarriors. It should not be difficult, let's see...
```

Comment:
```suggestion
After selecting the language, you are presented with your very first challenge. It's an entry level task which you need to solve to prove your dedication and commitment to learning.
```

"It should not be difficult" was already said with "an entry level task".

We can omit "prove yourself worthy of training among CodeWarriors" since it can be misinterpreted as unwelcoming and `codewarrior` is a new term.

By the way, we actually don't really like the term `Codewarrior`. Jake told me one of the early contributors named it and he doesn't like it. We should probably stop using it or reduce its usage.

Reply:
Apparently you haven;t tried to solve that 8 kyu AGDA kata :)

I thought that "entry level" != "easy", especially for newbies, and I wanted to sound encouraging. But if you say so, than ok 👍